bitwarden_license/test/Commercial.Core.Test/SecretsManager/AccessPolicies/CreateAccessPoliciesCommandTests.cs

```diff
@@ -17,9 +17,127 @@ namespace Bit.Commercial.Core.Test.SecretsManager.AccessPolicies;
 [ProjectCustomize]
 public class CreateAccessPoliciesCommandTests
 {
+    private static List<BaseAccessPolicy> MakeDuplicate(List<BaseAccessPolicy> data, AccessPolicyType accessPolicyType)
```

Comment:
Do you actually need to create a duplicate? Since we have to mock that endpoint.

Reply:
There is no endpoint being mock in the duplicate check `CheckForDistinctAccessPolicies`.

We do need to construct duplicates to pass into the function for the check to trigger.